### ISSUE TITLE
Missing warning about whitespaces around operators

### ISSUE THREAD

**Issue by dasm** [PyCQA/pycodestyle#225]
According to PEP8, user should consider adding whitespace around the operators. PEP8 doesn't detect this situation, and doesn't give any warnings.

user@server:~/$ cat example.py 
def function():
    varname = 1
    varname2 = varname+1

```
def showError ():
    pass
```

user@server:~/$ pep8 example.py -v
local configuration: in ~/$
checking example.py
example.py:5:18: E211 whitespace before '('
